### PR TITLE
Minor meta updates

### PR DESCRIPTION
**Pull request by Tyil**
This PR just applies some minor updates, mostly to metafiles.

The most important aspect however is the `license` field in the `META6.json`. As of right now there is no license available at all, which makes it non-free software. I've set it as `GPL-3.0` in the `META6.json` as this is my preferred license, but any free license should work.

### CODE REVIEW

File: META6.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        "Sem Schilder <sem@tropical.email>"
+    ],
+    "tags": [
+        "linux",
+        "system",
+        "unix"
+    ],
+    "test-depends": [
+    ],
+    "source-url": "https://github.com/the-eater/shinit"
```
Comment (the-eater) [inline META6.json]: missing `.git`

File: META6.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+    "perl": "6",
+    "name": "shinit",
+    "version": "0.0.1",
+    "auth": "the-eater:shinit",
```
Comment (the-eater) [inline META6.json]: Wasn't it `github:the-eater`?

**Comment by the-eater** [comment]
:+1: -comments

**Review by the-eater — CHANGES_REQUESTED** [review]

**Comment by Tyil** [comment]
Updated the `META6.json` as per your request.

**Comment by the-eater** [comment]
Thanks :)

**Pull request merged**